docker/pytest/requirements.in

```diff
@@ -4,3 +4,4 @@ requests==2.25.1
 jsonschema<5
 junitparser==2.8.0
 pexpect==4.8.0
+six==1.12.0
```

Comment:
I will take your word on that we still need `six`. Would be good to make a separate bug somewhere to gradually start removing references though it - in a Py3-only codebase it shouldn't be needed.

Reply:
It's a dist package on our debian 10 based image. On this ubuntu image it's not provided so need to be listed.

But yes, as soon as we rid the codebase of six it should go here too. It's only needed because some file included it.